README.md

```diff
@@ -274,6 +274,16 @@ The current values for these are available as secure notes in 1Password for each
   although this could be a potential future option - the code we inherited from crime apply does have the capability of loading 
   a Portal metadata file from a URL.
 
+## Manual Puppeteer upgrade
+
+The application uses puppeteer as part of its testing pipeline - namely as part of the browser tools dockerfile. This is pinned to a specific puppeteer version, but because Chrome updates quite regularly, we have to manually update this when a new release comes out. 
+
+Here is an example PR of what the update looks like: https://github.com/ministryofjustice/laa-check-client-qualifies/pull/1482/files
```

Comment:
do we want to add steps, something like:

- create an appropriately named branch from main, to reference the updated puppeteer version , ie. `puppeteer-22**`
- update `browser_tools_docker_image.yml` & `package.json` with updated puppeteer version
- run `yarn install` to update `yarn.lock`
- update `.circleci/config.yml` & `.github/workflows/browser_tools_docker_image.yml` to reference your appropriately named branch